src/Serilog/Core/LevelOverrideMap.cs

```diff
@@ -58,18 +58,15 @@ public LevelOverride(string context, LoggingLevelSwitch levelSwitch)
     }
 
     public void GetEffectiveLevel(
-#if FEATURE_SPAN
         ReadOnlySpan<char> context,
```

Comment:
I do not understand why span is used here. At the first glance it can be safely changed to string.